docs/badges.html

```diff
@@ -64,6 +64,10 @@
 
   clipboard.on('success', function(e) {
     e.trigger.innerHTML = "Copied!"
+
+    setTimeout(function() {
+      e.trigger.innerHTML = "Copy"
+    }, 1000);
```

Comment:
you need to clear the timer in case of multiple clicks.

```js
var timer = null;
clipboard.on('success', function(e) {
  e.trigger.innerHTML = "Copied!"
  if (timer) {
    clearTimeout(timer);
  }
  timer = setTimeout(function() {
    e.trigger.innerHTML = "Copy"
  }, 1000);
});
```